scripts/zones/Uleguerand_Range/IDs.lua

```diff
@@ -17,6 +17,9 @@ zones[dsp.zone.ULEGUERAND_RANGE] =
         KEYITEM_OBTAINED        = 6404, -- Obtained key item: <keyitem>.
         NOTHING_OUT_OF_ORDINARY = 6415, -- There is nothing out of the ordinary here.
         CONQUEST_BASE           = 7062, -- Tallying conquest results...
+        SOMETHING_GLITTERING    = 7333, -- You see something glittering beneath the surface of the ice. There are many cold Moblin Hotroks scattered around the area.
+        WHAT_LIES_BENEATH       = 7334, -- Could someone be trying to melt the ice to retrieve what lies beneath?
```

Comment:
This still isn't correct.  It should be exactly as quoted.  This is what's in the POL extract:

```
  <thing type="DialogTableEntry">
    <field name="index">7333</field>
    <field name="text">You see something glittering beneath the surface of the ice.≺Prompt≻
</field>
  </thing>
  <thing type="DialogTableEntry">
    <field name="index">7334</field>
    <field name="text">There are many cold ≺Possible Special Code: 01≻≺Possible Special Code: 05≻%ａ≺BAD CHAR: 80≻≺BAD CHAR: 80≻ scattered around the area.
Could someone be trying to melt the ice to retrieve what lies beneath?≺Prompt≻
</field>
```

For message 7333 we drop the <Prompt>.

For message 7334 we replace the Special Code crap with "<item>", and replace the line break with a space, and drop the <Prompt>